draft-dt-tls-external-psk-guidance.md

```diff
@@ -168,48 +166,18 @@ The attack proceeds as follows:
 
-This attack violates the peer authentication property, and if `C` supports a
-weaker set of cipher suites than `B`, this attack also violates the downgrade
-protection property.  This rerouting is a type of identity misbinding attack
-{{Krawczyk}}{{Sethi}}.  Selfie attack {{Selfie}} is a special case of the
-rerouting attack against a group member that can act both as TLS server and
-client. In the Selfie attack, a malicious non-member reroutes a connection
-from the client to the server on the same endpoint.
-
-Entropy properties of external PSKs may also affect TLS security properties.
-In particular, if a high entropy PSK is used, then PSK-only key establishment
-modes are secure against both active and passive attack. However, they lack
-forward security. Forward security may be achieved by using a PSK-DH mode.
-
-In contrast, if a low entropy PSK is used, then PSK-only key establishment
-modes are subject to passive exhaustive search passive attacks which
-will reveal the traffic keys. PSK-DH modes are subject to active
-attacks in which the attacker impersonates one side. The exhaustive
-search phase of these attacks can be mounted offline if the attacker
-captures a single handshake using the PSK, but those attacks will
-not lead to compromise of the traffic keys for that connection
-because those also depend on the Diffie-Hellman (DH) exchange. Low
-entropy keys are only secure against active attack if a PAKE is used with TLS.
+This attack violates the peer authentication property, and if `C` supports a weaker set of cipher suites than `B`, this
+attack also violates the downgrade protection property. This rerouting is a type of identity misbinding attack
+{{Krawczyk}}{{Sethi}}. Selfie attack {{Selfie}} is a special case of the rerouting attack against a group member that
+can act both as TLS server and client. In the Selfie attack, a malicious non-member reroutes a connection from the
+client to the server on the same endpoint.
 
-# Recommendations for External PSK Usage
+Entropy properties of external PSKs may also affect TLS security properties. In particular, if a high entropy PSK is used, then PSK-only key establishment modes are secure against both active and passive attack. However, they lack
+forward security. Forward security may be achieved by using a PSK-DH mode.
+ 
+In contrast, if a low entropy PSK is used, then PSK-only key establishment modes are subject to passive exhaustive search passive attacks which will reveal the traffic keys. PSK-DH modes are subject to active attacks in which the attacker impersonates one side. The exhaustive search phase of these attacks can be mounted offline if the attacker captures a single handshake using the PSK, but those attacks will
+not lead to compromise of the traffic keys for that connection because those also depend on the Diffie-Hellman (DH) exchange. Low entropy keys are only secure against active attack if a PAKE is used with TLS.
```

Comment:
This whole section section seems to have different formatting. I'd prefer we just revert this entirely. (That is, break lines at ~80-120 columns, give or take.) I can do that after we merge though.

Reply:
Yes. My editor doesn't add line breaks and I seem to like it. I guess it is best if you do it after the merge.